.github/workflows/upload-conan-deps.yml

```diff
@@ -21,7 +21,6 @@ on:
       - .github/workflows/upload-conan-deps.yml
 
       - .github/workflows/reusable-strategy-matrix.yml
```

Comment:
`reusable-strategy-matrix` and `setup-conan` should have been included in the file lists for `on-pr` and `on-trigger`, since they also depend on it.